<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginModeButtons</class>
 <widget class="QFrame" name="CLoginModeButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login mode</string>
  </property>
  <property name="title" stdset="0">
   <string/>
  </property>
  <property name="flat" stdset="0">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="hl_LoginMode">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item alignment="Qt::AlignRight">
    <widget class="QRadioButton" name="rb_LoginNormal">
     <property name="text">
      <string>Normal</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QRadioButton" name="rb_LoginStealth">
     <property name="text">
      <string>Stealth</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QRadioButton" name="rb_LoginObserver">
     <property name="text">
      <string>Observer (co-pilot)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_NoSimulator">
     <property name="text">
      <string>No simulator!</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="hs_LoginModeButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
